content/auth/capabilities.textile

```diff
@@ -0,0 +1,194 @@
+
+You can also have a resource name of @[*]*@, which will match all queues, all metachannels, and all channels.
+
+Wildcards are also supported for "operations":#capability-operations, by requesting an operations list of @['*']@.
+
+h2(#capability-operations). Capability operations
+
+The following capability operations are available for API keys and issued tokens:
+
+- subscribe := can subscribe to messages and presence state change messages on channels, and get the presence set of a channel
+- publish := can publish messages to channels
+- presence := can register presence on a channel (enter, update and leave)
+- history := can retrieve message and presence state history on channels
+- stats := can retrieve current and historical usage statistics for an app
+- push-subscribe := can subscribe devices for push notifications
+- push-admin := can manage device registrations and push subscriptions for all devices in an app
+- channel-metadata := can get metadata for a channel, and enumerate channels
+
+While most of these capabilities need to be enabled for the resource you're using them with, there are exceptions. The @stats@ permission only does something when attached to the wildcard resource @'*'@, or a resource that contains that as a subset, such as @'[*]*'@, since stats are app-wide. 
+
+The @channel-metadata@ permission works both ways. When associated with a specific channel or set of channels it allows you to use the "Channel Status API":/rest/channel-status#channel-status to request the status of that channel. When associated with the wildcard resource @'*'@ it takes on an additional meaning: as well as allowing channel status requests for all channels, it also allows you to "enumerate all active channels":/rest/channel-status#enumeration-rest. 
+
+<!-- CAVEAT-INPUT: Channel mode flags -->
+"Channel mode flags":LINKME enable a client to specify a subset of the capabilities granted by their token or API key as channel options. 
+
+Channel mode flags offer the ability for clients to use different capabilities for different channels, however, as they are flags and not permissions, they cannot be enforced by an authentication server. Channel mode flags also enable clients to be present on a channel without subscribing to presence events.
+
+h2(#capabilities-key). API key capabilities
+
+An "Ably API key":/auth#api-key can have single set of permissions, applied to any number of "channels":/LINKME or "queues":/general/queues.
+
+You can also choose whether to restrict the API key to only channels, only "queues":/general/queues, or to match a set of channel or queue names. If you've chosen to restrict the API key to *selected channels and queues*, you can use a comma separated list of resources the API key can access, making use of wildcard to provide access to areas of your app. It is worth noting an API key will provide the same permissions to all resources it has access to.
```

Comment:
"making use of wildcard to provide"

 making use of wildcard**s** to provide

Reply:
Fixed, thanks.